chapters/teams.Rmd

```diff
@@ -1349,7 +1362,7 @@ if someone's name appears in the Git history for a project,
 they are listed as a co-author on papers for that project.
 A new member of your team has complained that this is unfair:
 people who haven't contributed for over two years are still being included as authors,
-while they aren't even though they have spent many hours doing code reviews of others' work.
+while they aren't included because their contributions of doing many hours of code reviews don't show up in the Git history.
```

Comment:
"because the many hours they have spent doing code reviews don't show up in the Git history"